src/Tizen.NUI/src/public/XamlBinding/BindableObject.cs

```diff
@@ -554,7 +554,13 @@ bool CanBeSetFromStyle(BindableProperty property)
             return false;
         }
 
-        internal void SetDynamicResource(BindableProperty property, string key)
+        /// <summary>
+        /// Internal use, will never open
+        /// </summary>
+        /// <param name="property"></param>
+        /// <param name="key"></param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void SetDynamicResource(BindableProperty property, string key)
```

Comment:
Could you let us know why this method should change to public one?
Also, it'd better write a brief summary of this API.